shielddash/core/static/js/app/components/views/fetch-error.js

```diff
@@ -5,7 +5,11 @@ export default function(props) {
     <dl className="component-error-message">
       <dt>{props.status}</dt>
       <dd>There was a problem retrieving data from the server. You may
-      want to try signing out then signing in again.</dd>
+      want to try signing out then signing in again.
+      <p>
+        The server returned the following details: {props.detail}
+      </p>
+      </dd>
```

Comment:
Just a small nitpick. Since the first part of the text (*There was a problem... signing in again.*) is also a self-contained block of text, we could also wrap that in a `<p>`.